modules/tinymce/src/plugins/lists/test/ts/webdriver/ContentEditableFalseTest.ts

```diff
@@ -0,0 +1,96 @@
+import { RealKeys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/lists/Plugin';
+
+describe('webdriver.tinymce.plugins.lists.BackspaceDeleteTest', () => {
```

Comment:
This is doing more than just backspace and delete (I guess the file moved too).

I don't see any test for the toolbar button list actions?

Reply:
Missed when writing the test, I copied the content.